.idea/misc.xml

```diff
@@ -3,6 +3,15 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="../../../../../layout/compose-model-1633680131221.xml" value="0.27280405405405406" />
```

Comment:
Untrack files changes 
https://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/